test/parse-result.js

```diff
@@ -220,6 +220,10 @@ describe('Parse result namespace', () => {
       expect(sourceMaps.first().toValue()).to.deep.equal([[1, 2]]);
     });
 
+    it('should have a convenience method for retrieving source map', () => {
+      expect(element.sourceMap.first().toValue()).to.deep.equal([[1, 2]]);
```

Comment:
> We could perhaps provide some other convenience method on top offering the underlying data for the use-cases.

That is what this method is for. If they want the original thing, they can do `attributes.sourceMap`.